include/c2pa.h

```diff
@@ -135,15 +213,284 @@ char *c2pa_sign_file(const char *source_path,
                      const char *data_dir);
 
 /**
- * Releases a string allocated by Rust
+ * Frees a string allocated by Rust
  *
  * # Safety
  * Reads from null terminated C strings
  * The string must not have been modified in C
- * can only be released once and is invalid after this call
+ * can only be freed once and is invalid after this call
+ */
+IMPORT extern void c2pa_string_free(char *s);
+
+/**
+ * Frees a string allocated by Rust
+ *
+ * Provided for backward api compatibility
```

Comment:
Perhaps add a stronger deprecation notice here? Otherwise, I'm spending time figuring out the difference btw free and release.